src/common/utxobased/db/newProcessor.ts

```diff
@@ -151,20 +151,30 @@ export async function makeNewProcessor(
       baselets = await makeBaselets({ disklet })
     },
 
-    async insertUtxo(_utxo: IUTXO): Promise<void> {
-      return
+    async insertUtxo(utxo: IUTXO): Promise<void> {
+      return await baselets.utxo(async tables => {
+        await tables.utxoById.insert('', utxo.id, utxo)
+      })
     },
 
-    async removeUtxos(_utxoIds: string[]): Promise<void> {
-      return
+    async removeUtxos(utxoIds: string[]): Promise<void> {
+      return await baselets.utxo(async tables => {
+        await tables.utxoById.delete('', utxoIds)
+      })
     },
 
-    async updateUtxo(_utxo: IUTXO): Promise<void> {
-      return
+    async updateUtxo(utxo: IUTXO): Promise<void> {
+      return await baselets.utxo(async tables => {
+        await tables.utxoById.insert('', utxo.id, utxo)
+      })
     },
 
-    async fetchUtxos(_utxoIds: string[]): Promise<IUTXO[]> {
-      return []
+    async fetchUtxos(utxoIds: string[]): Promise<IUTXO[]> {
+      if (utxoIds.length === 0) return []
```

Comment:
I just noticed, that in this case, we should be returning all UTXO's. It's also already documented in the interface declaration: `  // fetch either all UTXOs if the array is empty or as selected from an array
  // of UTXO ids`

Reply:
I had to use `dumpData` to do this because `query` for `HashBase` wont allow an empty array of hashes.